src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -542,8 +536,8 @@ void hiopVectorRajaPar::copyToStarting(int start_index, hiopVector& dst) const
   if(v.n_local_ == 0)
     return;
 
-  auto& rm = umpire::ResourceManager::getInstance();
-  rm.copy(v.data_dev_, this->data_dev_ + start_index, v.n_local_*sizeof(double));
+  //TODO: pointer arithmetic on host should be avoided
+  v.exec_space_.copy(v.data_dev_, this->data_dev_ + start_index, v.n_local_, exec_space_);
```

Comment:
Again, this hasn't been an issue before, and assume pre/post conditions are validated this should be fine?